app/src/main/res/layout-land/profile_chooser_add_view.xml

```diff
@@ -11,29 +11,34 @@
       type="androidx.databinding.ObservableField&lt;Boolean&gt;" />
   </data>
 
-  <LinearLayout
+  <androidx.constraintlayout.widget.ConstraintLayout
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:orientation="horizontal">
+    app:layout_constraintHorizontal_chainStyle="packed">
 
     <View
       android:id="@+id/add_profile_divider_view"
       android:layout_width="0.5dp"
       android:layout_height="match_parent"
       android:background="@color/oppiaProfileChooserDivider"
-      android:visibility="@{hasProfileEverBeenAddedValue ? View.GONE : View.VISIBLE}" />
+      android:visibility="@{hasProfileEverBeenAddedValue ? View.GONE : View.VISIBLE}"
+      app:layout_constraintEnd_toStartOf="@+id/add_profile_item"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toTopOf="parent" />
 
     <LinearLayout
       android:id="@+id/add_profile_item"
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
       android:layout_marginStart="16dp"
       android:layout_marginEnd="16dp"
-      android:layout_marginBottom="24dp"
```

Comment:
@ARJUPTA Maybe removal of this margin is creating the issues. You should not change the padding/margin in this PR.

Reply:
Thanks, but I think this is the only time I deleted a margin without replacement. As this file is going to be used for Landscape version then how is that the portrait mode design is also being affected

Reply:
@ARJUPTA Yeah you are right. I haven't investigated the issue you are facing. I will defer to @MohamedMedhat1998 for that.

@MohamedMedhat1998 Can you review and provide update on this. And if needed then assign it to me.